SPECS/dracut/dracut.spec

```diff
@@ -211,6 +211,10 @@ ln -sr %{buildroot}%{_bindir}/dracut %{buildroot}%{_sbindir}/dracut
 %dir %{_sharedstatedir}/dracut/overlay
 
 %changelog
+* Tue Apr 02 2024 Lanze Liu <lanzeliu@microsoft.com> - 055-9
+- Cherry-picked Overlay Dracut Module changes from 3.0-dev to main.
+- PRs picked: #8332, #8481, #8597
```

Comment:
Could you also summarize the changes? It may help to have pointers to the cherry-pick origin, but explaining what the changes are here may save people trouble. Also, RPM changelog can be read on a running image, where these comment's won't be too helpful.

Reply:
Fixed, thanks.